packages/venia-concept/src/components/Checkbox/checkbox.css

```diff
@@ -1,13 +1,56 @@
 .root {
-    padding: 0.5rem 1.5rem;
-    font-size: 0.85rem;
-    outline: none;
+    align-items: center;
+    color: rgb(var(--venia-text));
+    display: inline-grid;
+    font-size: 1rem;
+    font-weight: 400;
+    gap: 0.5rem;
+    grid-auto-flow: column;
+    grid-template-columns: auto min-content;
+    justify-items: center;
+    margin: 0.5rem;
+}
+
+.root:active,
+.root:focus-within {
```

Comment:
The `focus-within` is a really nice visual touch 👍

Reply:
Yeah, it's a great tool. After some more consideration, though, we've decided to go with a different set of focus styles.